operation.go

```diff
@@ -516,9 +522,16 @@ func (operation *Operation) ParseRouterComment(commentLine string) error {
 	path := matches[1]
 	httpMethod := matches[2]
 
-	operation.Path = path
-	operation.HTTPMethod = strings.ToUpper(httpMethod)
+	signature := RouteProperties{
+		Path:       path,
+		HTTPMethod: strings.ToUpper(httpMethod),
+	}
+	if operation.RouterProperties[0].IsDefault {
+		operation.RouterProperties = []RouteProperties{signature}
+		return nil
+	}
 
```

Comment:
So you added the isDefault property instead of using len(operation.RouterProperties)?

Reply:
I also thought about that initially. But, If I add
` if len(operation.RouterProperties) == 1 && (compare the values of HTTPMethod, Path with default) {
		//override the default
	}`
We won't be able to differentiate in cases when a user explicitly adds the default values in comments and would end up overriding the user entered values.

I can think of another way, if we remove the default initialization
from https://github.com/pc-coder/swag/blob/master/operation.go#L64
and check for len == 0 at https://github.com/pc-coder/swag/blob/master/parser.go#L570 and then add defaults, we can get rid of IsDefault.

Thoughts?

Reply:
you can initialize operation with an empty slice and simply append the slice unconditionaly
```
result := &Operation{
                ..... snip
		RouterProperties: []RouteProperties{},
}
```

```
// ParseRouterComment parses comment for gived `router` comment string.
func (operation *Operation) ParseRouterComment(commentLine string) error {
	var matches []string

	if matches = routerPattern.FindStringSubmatch(commentLine); len(matches) != 3 {
		return fmt.Errorf("can not parse router comment \"%s\"", commentLine)
	}
	path := matches[1]
	httpMethod := matches[2]

	signature := RouteProperties{
		Path:       path,
		HTTPMethod: strings.ToUpper(httpMethod),
	}

	operation.RouterProperties = append(operation.RouterProperties, signature)

	return nil
}
```

Reply:
Have made the requested changes.

I thought removing default initialization would be a breaking change.